bin/cd_sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Script for use from Makefiles to change to a directory and execute a command
+# For example: cd_sh subdir "cmd1 arg1 && cmd2 arg2"
```

Comment:
Is there a benefit compared to doing
```
(cd subdir && cmd1 arg1 && cmd2 arg2)
```
directly?

Reply:
See the comment above, the benefit is only the entering/exiting messages